whats_left.py

```diff
@@ -483,11 +484,17 @@ def remove_one_indent(s):
 if args.signature:
     print("\n# mismatching signatures (warnings)")
     for modname, mismatched in result["mismatched_items"].items():
-        for (item, rustpy_value, cpython_value) in mismatched:
-            if cpython_value == "ValueError('no signature found')":
-                continue # these items will never match
-
-            print(f"{item} {rustpy_value} != {cpython_value}")
+        for i, (item, rustpy_value, cpython_value) in enumerate(mismatched):
+            if cpython_value and cpython_value.startswith("ValueError("):
+                continue  # these items will never match
+            if rustpy_value is None or rustpy_value.startswith("ValueError("):
+                rustpy_value = f" {rustpy_value}"
+            print(f"{item}{rustpy_value}")
+            if cpython_value is None or cpython_value.startswith("ValueError("):
+                cpython_value = f" {cpython_value}"
+            print(f"{' ' * len(item)}{cpython_value}")
+            if i < len(mismatched) - 1:
+                print()
```

Comment:
Thanks! Can you add a comment explaining your logic here? Are we filtering the same things here? (I.e do we ever have `ValueError("something else")` which is now being ignored?

Reply:
I'm not at my computer right now, you can remove the if statement and run the code to see the error that I'm filtering. The point is really that if we can't get the signature for a function in CPython then there's no point in checking it signature in RustPython.